analysis_options.yaml

```diff
@@ -3,6 +3,10 @@ include: package:flutter_lints/flutter.yaml
 analyzer:
   exclude: [lib/generated_plugin_registrant.dart]
 
+language:
```

Comment:
Please explain these changes.

Reply:
These ensure that code specifies generic type arguments and make sure that code is maintainable with improved performance. They were by default mentioned in many analysis files, so I thought of using them here too. You may want to read [this](https://github.com/dart-lang/language/blob/master/resources/type-system/strict-raw-types.md) to learn more.

Reply:
How it fixes the issue #149 ? Your aim for this PR is to fix #149 or to do something irrelevant ?

Reply:
It tries to add more customary linting checks and analysis options for a better analysis, that's why I mentioned partially fixes 😅